internal/ethapi/api.go

```diff
@@ -1020,6 +1020,10 @@ func (s *BlockChainAPI) Call(ctx context.Context, args TransactionArgs, blockNrO
 }
 
 func DoEstimateGas(ctx context.Context, b Backend, args TransactionArgs, blockNrOrHash rpc.BlockNumberOrHash, gasCap uint64) (hexutil.Uint64, error) {
+	// If there is no data, it is considered a simple transfer which has a fixed cost.
+	if args.To != nil && len(args.data()) == 0 {
```

Comment:
Couldn't this break fallback functions that have logic in them? An [example](https://optimistic.etherscan.io/tx/0xb967026f9672b8ae2be7c7dbb8c9485537cc8013f84c43b183c003750b95bd36) here